app/mailers/rent_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class RentMailer < ApplicationMailer
+  def new_rent_send(rent_id)
+    rent = Rent.find(rent_id)
+    email = rent.user.email
+    book = rent.book
+
+    @date = Time.zone.today
```

Comment:
you don't need to create all of these variables, just find the rent and make it an instance variable to be accessed on `new_rent_send.erb` and calls every data through this.

Reply:
Done